Migrate Thread usage instructions (#37)
canonical/matter-docs: how-to/otbr-on-ubuntu.rst

.. code:: bash

    snap logs -n 100 -f openthread-border-router

Form a Thread network
---------------------

Use the CTL tool to initialize the Thread network:

.. code:: bash

    sudo openthread-border-router.ot-ctl dataset init new
    sudo openthread-border-router.ot-ctl dataset commit active
    sudo openthread-border-router.ot-ctl ifconfig up
    sudo openthread-border-router.ot-ctl thread start

> Alternatively, these steps could be performed with the GUI at https://localhost.

Reviewer: ```suggestion
Alternatively, these steps could be performed with the GUI at [http://localhost:80](http://localhost:80).
```

Please refer to the instructions `here <https://openthread.io/guides/border-router/web-gui.md>`_ to configure and form, join, or check the status of a Thread network using the GUI.

Controlling a Thread device
---------------------------

> To commission and control a Matter Thread device, you can use Chip Tool.
> Read :doc:`here <chip-tool-commission-and-control>` how to do that.

Reviewer: ```suggestion
To commission and control a Matter Thread device, you can use Chip Tool; refer to :doc:`chip-tool-commission-and-control`.
```
